app/src/main/res/layout/fragment_ai_proposition.xml

```diff
@@ -24,19 +24,27 @@
     android:orientation="vertical"
     android:theme="@style/AiCursorAndPrimaryColorTheme">
 
-    <com.google.android.material.appbar.MaterialToolbar
-        android:id="@+id/toolbar"
+    <com.google.android.material.appbar.AppBarLayout
+        android:id="@+id/appBarLayout"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
+        android:backgroundTint="@color/backgroundColor"
```

Comment:
I just realized we have a `toolbarLoweredColor` color we can use instead of a plain `backgroundColor`